papers/david_dotson/david_dotson.rst

```diff
@@ -324,18 +324,19 @@ We can add categories to individual Treants:
     # for all members
     >>> b.categories.add({'plant': 'tree'})
 
-We can access categories for an individual member:
+and we can access categories for individual Treants:
```

Comment:
I like treating code as part of the text (similar to equations, which should have punctuation and be part of a sentence IMHO)